front/lib/swr.ts

```diff
@@ -823,6 +827,45 @@ export function useAgentConfigurations({
   };
 }
 
+export function useProgressiveAgentConfigurations({
+  workspaceId,
+}: {
+  workspaceId: string;
+}) {
+  const {
+    agentConfigurations: initialAgentConfigurations,
+    isAgentConfigurationsLoading: isInitialAgentConfigurationsLoading,
+  } = useAgentConfigurations({
+    workspaceId,
+    agentsGetView: "assistants-search",
+    limit: 24,
+    includes: ["usage"],
+  });
+
+  const {
+    agentConfigurations: agentConfigurationsWithAuthors,
+    isAgentConfigurationsLoading: isAgentConfigurationsWithAuthorsLoading,
+    mutateAgentConfigurations,
+  } = useAgentConfigurations({
+    workspaceId,
+    agentsGetView: "assistants-search",
+    includes: ["authors", "usage"],
+  });
+
+  const isLoading =
```

Comment:
did you check that we don't sometimes need to distinguish the partially loaded state vs fully loaded state in the UI?

Reply:
AFAICT, the `AssistantBrowser` component does not surface any difference, I refined the type here: https://github.com/dust-tt/dust/pull/5766/files#diff-b1d3cf33fe34d9f4ddaf756f2d7340bc783959d9ecc6624e4032fdc7441fa38fL32.